### ISSUE TITLE
Can you give me some advice to implement the LO-Net?

### ISSUE THREAD

**Issue by dlfksj** [PythonLidar/pytorch-LONet#2]
Hi, I'm trying to implement the LO-Net with the Tensorflow.
I made the network but mine is not working.
I want to know does your network work well like the LO-Net paper.

My problem is, the network always give the same output regardless of input lidar scans.
I think vanishing gradient or dying ReLU would be the problem.

I want to get some advice if your network trained well.
Did you have the same issues like this before? What do you think would be the problem?

And can you let me know what initialization method should I use for the network?
What activation functions did you used for three fully connected layers at the end?
Or do I need some pre-processing for the input data like smoothing?

Thanks.
